proxy_http.go

```diff
@@ -200,8 +213,6 @@ func prepareRequest(req *http.Request) *http.Request {
 	req.URL.Scheme = "http"
 	// We need to make sure the host is defined in the URL (not the actual URI)
 	req.URL.Host = req.Host
-	req.URL.RawQuery = req.URL.RawQuery
```

Comment:
Why do we no longer copy the `RawQuery` here anymore? Is it no longer needed?

Reply:
It was just something the linter complained about. I don't think we need to copy it for any reason.

Reply:
Ok, thanks! This LGTM otherwise.